ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2527,3 +2527,34 @@ def test_tensorflow_in_top_k(
         pred=x[1],
         k=k,
     )
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.math.cumulative_logsumexp",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float", full=True),
+        valid_axis=True,
+        force_int_axis=True,
+        min_num_dims=1,
+        min_value=0.1,
+        max_value=10,
+    ),
+    exclusive=st.booleans(),
+    reverse=st.booleans(),
+)
+def test_tensorflow_cumulative_logsumexp(
+    *, dtype_x_axis, frontend, test_flags, fn_tree, on_device, exclusive, reverse
+):
+    input_dtype, x, axis = dtype_x_axis
+    assume(input_dtype != "bfloat16")
```

Comment:
You dont need to add any extra logic to filter out specific `dtypes` here. You can instead use the `@with_unsupported_dtypes` decorator in the `cumulative_logsumexp` function itself to achieve this purpose. You can read more about the use of this decorator [here](https://unify.ai/docs/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types)